packages/shared/src/components/squads/SharePostBar.tsx

```diff
@@ -59,48 +62,59 @@ function SharePostBar({
     <form
       onSubmit={onSubmit}
       className={classNames(
-        'flex flex-wrap items-center rounded-16 typo-callout border overflow-hidden',
+        'flex flex-col tablet:flex-row items-center rounded-16 typo-callout border overflow-hidden',
         'bg-theme-float focus-within:border-theme-divider-primary border-theme-divider-tertiary hover:border-theme-divider-primary',
         className,
       )}
     >
-      <ProfilePicture
-        className="order-1 m-3"
-        user={user}
-        size="large"
-        nativeLazyLoading
-      />
-      <input
-        type="url"
-        autoComplete="off"
-        name="share-post-bar"
-        className="flex-1 order-2 pl-1 w-24 mobileL:w-auto outline-none bg-theme-bg-transparent text-theme-label-primary focus:placeholder-theme-label-quaternary hover:placeholder-theme-label-primary typo-callout"
-        placeholder="Enter link to share"
-        onInput={(e) => setUrl(e.currentTarget.value)}
-        value={url}
-      />
-      {!url && (
-        <Button
-          type="button"
-          onClick={() => onNewSquadPost()}
-          buttonSize={sidebarRendered ? ButtonSize.Small : ButtonSize.Medium}
+      <span className="flex relative flex-row items-center w-full">
+        <ProfilePicture
+          className="m-3"
+          user={user}
+          size="large"
+          nativeLazyLoading
+        />
+        <input
+          ref={urlRef}
```

Comment:
Why do we need `urlRef` now vs before?

Reply:
Ah, it seems to be not used anymore, I was working on something have probably decided not to purse, will remove this one 🚀